analyze_hc_restore.pl

```diff
@@ -63,8 +63,9 @@ sub usage
   print "-d | --gpu-devices VALUE         Set the --gpu-devices to the comma-separated list given by VALUE\n";
   print "-O | --optimized-kernel-enable   Set the --optimized-kernel-enable command line switch\n";
   print "-w | --workload-profile NUM      Set the workload profile to NUM; select between the performance profiles 1 (reduced), 2 (default) or 3 (tuned)\n";
-  print "-n | --gpu-accel NUM             Set the gpu acceleration to NUM. 1, 8, 40, 80, 160\n";
-  print "-u | --gpu-loops NUM             Set the gpu loops to NUM. 8 - 1024\n";
+  print "-n | --kernel-accel NUM          Set the --kernel-acceleration to NUM. 1, 8, 40, 80, 160\n";
```

Comment:
maybe we should just wrtite
`Set the --kernel-accel to NUM. 1, 8, 40, 80, 160\n`
because `--kernel-acceleration` isn't really a valid command line option for hashcat... we shouldn't confuse the user

Reply:
That's a great point and my apologies for not even realizing that, I shouldn't  have simply done a word replacement thinking it would be sufficient. 
Inadvertently, your mention of confusing the user made me realize some additional inconsistencies with the latest release that I feel somewhat  compelled to address but will hold off for another patch when time permits and will address just his one for now.

Just as a note for myself for future reference:
`-t` and `-T` have been disassociated from their long expression option whereas `-T` now defines `--kernel-threads` and `-t` now sets `--markov-threshold` while `--gpu-temp-disable` has been succeeded by `--hwmon-disable` and `--runtime` must be set by the long expression.

Reply:
Resolved in commit [6e249ed](https://github.com/philsmd/analyze_hc_restore/commit/6e249ed27908bd07124e48656a0e1ebace266da9)